pause/Makefile

```diff
@@ -2,7 +2,8 @@ src = $(wildcard *.c)
 obj = $(src:.c=.o)
 
 override LIBS +=
-override CFLAGS += -std=c99 -Os -Wall -Wextra -static
+CFLAGS ?= std=c99 -Os -Wall -Wextra
```

Comment:
Should be
```
+CFLAGS ?= -std=c99 -Os -Wall -Wextra
```

Reply:
> Should be
>
>    +CFLAGS ?= -std=c99 -Os -Wall -Wextra

Thanks.  Fixed with 07405178d → 57a735291.